.circleci/config.yml

```diff
@@ -334,6 +334,15 @@ jobs:
       - run: touch .watchmanconfig
 
       - run: yarn test-ios
+
+      - run:
+          name:  Generate RNTesterPods Workspace with frameworks
+          command: cd RNTester && pod install --verbose
+          environment:
+            USE_FRAMEWORKS: '1'
+      
+      - run: yarn test-ios
```

Comment:
This just adds a second test run with frameworks enabled to the `test_ios` job. @fkgozali would you prefer this to be a separate parallel job (such as `test_ios_frameworks`) or is it better here?

Reply:
Actually, I think a separate job probably is better. Let me know if you would like me to change it back.

Reply:
I'll have to defer to @hramos for the actual tradeoff discussion. To me, putting it in one place is easier to maintain (because it's basically testing exactly the same thing, just with framework output).

So for now, let's get this in as is and we can adjust it if needed in the future.